packages/@aws-cdk/custom-resources/lib/aws-custom-resource/aws-custom-resource.ts

```diff
@@ -130,6 +130,17 @@ export interface AwsSdkCall {
    * @default - return all data
    */
   readonly outputPath?: string;
+
+  /**
+   * Used for running the SDK calls in underlying lambda with a different role
+   * Can be used primarily for cross-account requests to for example connect
+   * hostedzone with a shared vpc
+   *
+   * Example for Route53 / associateVPCWithHostedZone
+   *
+   * @default - run without assuming role
+   */
+  readonly assumedRoleArn?: string;
```

Comment:
For consistency's sake this should be an `IRole`.

Reply:
Will that be serialized correctly as a RoleArn when sent to the lambda, or do the lambda then receive a IRole to get the RoleArn from?